en/websites_privacy_notice.md

```diff
@@ -16,7 +16,7 @@ We may receive personal information from you based on your interaction with us o
 
-* **Contributors**: Volunteering for Mozilla as a community contributor may require Mozilla and others to communicate with you at the email address that you provide in connection to your contribution and to recognize your efforts. If you contribute to Bugzilla, Mozilla Reps, or our code base, then your email address and possibly your name will be publicly available to all internet users. If you create a profile at Mozillians.org, it will be accessible to Mozilla employees and Mozilla contributors. You can edit your profile data at [Profile Settings](https://mozillians.org/user/edit). We sometimes use contributor information from sources (such as Bugzilla) in dashboards to visually share aggregated data on the Mozilla community. An example is [https://wiki.mozilla.org/Contribute/Dashboards](https://wiki.mozilla.org/Contribute/Dashboards). If possible, we try to minimize contact information that is publicly displayed.
+* **Contributors**: Volunteering for Mozilla as a community contributor may require Mozilla and others to communicate with you at the email address that you provide in connection to your contribution and to recognize your efforts. If you contribute to Bugzilla, Mozilla Reps, or our code base, then your email address and possibly your name will be publicly available to all internet users. If you create an account on Mozilla Community (platform powered by KhorosKhorus, which has itsit’s own privacy notice), your name, your avatar, your posts and replies and other information you share will be accessible to Mozilla and other community members. If you create an account on Mozilla Connect (platform powered by Khoros, [which has itsit’s own privacy notice](https://khoros.com/privacy)), your name, your avatar, your posts and replies and other information you share will be accessible to Mozilla and other community members. If you create a profile at people.mozilla.org, it will be accessible to Mozilla employees and Mozilla contributors. You can edit your profile data at [Profile Settings](https://mozillians.org/user/edit). We sometimes use contributor information from sources (such as Bugzilla) in dashboards to visually share aggregated data on the Mozilla community. An example is [https://wiki.mozilla.org/Contribute/Dashboards](https://wiki.mozilla.org/Contribute/Dashboards). If possible, we try to minimize contact information that is publicly displayed.
```

Comment:
Is `itsit` the correct word used in here or is it a typo? The only Its-it is an ice cream.  The usage of `itsit's` is even rarer.